site/e2e/helpers.ts

```diff
@@ -48,6 +48,10 @@ export function requiresLicense() {
 	test.skip(!license);
 }
 
+export function noPremiumLicense() {
```

Comment:
```suggestion
export function requiresUnlicensed() {
```

might jive a bit better with the existing `requiresLicense` helper